wordpress-mobile/WordPress-Android
# Update Bundle

This small PR updates `bundle` and the local `ruby` version to sync with the other projects.

To test:
1. Run `bundle install` and verify that it works without errors and that the repository is clean after that. 

PR submission checklist:
